olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -628,34 +628,30 @@ spec:
-                        value: "registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-receiver"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-receiver"
                       - name: "KAFKA_IMAGE_kafka-broker-dispatcher__kafka-broker-dispatcher"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-dispatcher"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-dispatcher"
+                      - name: "KAFKA_IMAGE_kafka-channel-receiver__kafka-channel-receiver"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-receiver"
+                      - name: "KAFKA_IMAGE_kafka-channel-dispatcher__kafka-channel-dispatcher"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-dispatcher"
                       - name: "KAFKA_IMAGE_kafka-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-kafka-controller"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-kafka-controller"
                       - name: "KAFKA_IMAGE_kafka-sink-receiver__kafka-sink-receiver"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-receiver"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-receiver"
                       - name: "KAFKA_IMAGE_kafka-source-dispatcher__kafka-source-dispatcher"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-dispatcher"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-dispatcher"
                       - name: "KAFKA_IMAGE_kafka-webhook-eventing__kafka-webhook-eventing"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-webhook-kafka"
-                      - name: "KAFKA_IMAGE_kafka-controller-post-install-__post-install"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-post-install"
-                      - name: "KAFKA_IMAGE_knative-kafka-storage-version-migrator-__migrate"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.2.3:knative-eventing-kafka-broker-webhook-kafka"
+                      - name: "KAFKA_IMAGE_kafka-controller-post-install__post-install"
+                        value: "docker.io/aliok/post-install-556eadd8a59c2213d960089c7a8fad4a@sha256:4fc202b0b094be7332616a87280af735dd57b7e610101dc9501e81489ca81aa0"
```

Comment:
This does not really fly :smile: 

I am even OK to merge this, but the relevant changes to that image do need backports for:
* 1.2
* and 1.3 (just that we do not forget this)

/assign @aliok